extensions/wikia/AdEngine3/src/startup.js

```diff
@@ -104,6 +105,12 @@ function startAdEngine(inhibitors) {
 		eventService.on(AdSlot.SLOT_RENDERED_EVENT, (slot) => {
 			slot.getElement().classList.remove('default-height');
 		});
+
+		eventService.communicator.actions$.pipe(
+			ofType(recirculationSwappedEvent)
```

Comment:
I think the event could be named more adequately to its role (informing about hidden popular pages)

Reply:
Renamed to `recirculationHiddenEvent`.